.pre-commit-config.yaml

```diff
@@ -11,8 +11,6 @@ repos:
     -   id: end-of-file-fixer
     -   id: requirements-txt-fixer
     -   id: trailing-whitespace
-    -   id: no-commit-to-branch
```

Comment:
There are alternative ways to protect specific branches from direct checkins using only Github settings.